examples/research_projects/jax-projects/README.md

```diff
@@ -0,0 +1,161 @@
+# Flax/JAX community week 🤗
+
+Welcome to the Flax/JAX community week! The goal of this week is to make compute-intensive NLP and CV projects (like pre-training BERT, GPT2, CLIP, ViT) 
+practicable for a wider audience of engineers and researchers. 
+To do so, we will try to teach **you** how to effectively use JAX/Flax on TPU and help you to complete a fun NLP and/or CV project in JAX/Flax during the community week. 
+
```

Comment:
I don't see any mention of free TPUs. Is that something we should bring up here?